CONTRIBUTING.md

```diff
@@ -68,60 +73,74 @@ For a complete look at this repository's labels, see the
-someone else.
+2. __Use the GitHub issue search__ &mdash; check if the issue you are encountering has
+already been reported by someone else.
 
-3. __Check if the issue has already been fixed__ &mdash; try to reproduce your issue using the latest commit to
-the `main` branch in the repository.
+3. __Check if the issue has already been fixed__ &mdash; try to reproduce your issue
+using the latest version of the Slurm charms.
 
-4. __Isolate the problem__ &mdash; the more pinpointed the issue is, the easier time the developers will have fixing it.
+4. __Isolate the problem__ &mdash; the more pinpointed the issue with the Slurm charms
+is, the easier it is to fix.
 
 A good bug report should not leave others needing to chase you for more information.
-Please try to be as detailed as possible in your report. What is your environment? What steps will reproduce the issue?
-What operating system are you experiencing the problem on? Have you had the same results on a different
-operating system? What would you expect to be the outcome? All these details will help the developers fix any
-potential bugs.
-
-## Feature Requests
-
-All feature requests should be posted to GitHub Discussions and tagged as `Type: Idea Bank`. The maintainers of
-the project already know the features they want to incorporate into it, but they are always open to
-new ideas and potential improvements. GitHub Discussions is the best place to post these types of requests
-because it allows for feedback from the entire community and does not bloat the issue tracker. Please note that not
-all feature requests will be incorporated into the code base. Also, feature requests posted on the issue tracker
-will be tagged as `Type: Invalid` and closed. Lastly, please note that spamming the maintainers to incorporate a
-feature you want into any of the charms will not improve its likelihood of being implemented; it may result in you receiving
-a temporary ban from the repository.
+Some common questions you should answer in your report include:
+
+* What is your current environment?
+* Were you able to reproduce the issue in another environment?
+* Which commands/actions/configuration options/etc produce the issue?
+* What was your expected outcome?
+
+Please try to be as detailed as possible in your report. All these details will help the
+maintainers quickly fix issues with the Slurm charms.
+
+## Enhancement Proposals
+
+The Charmed HPC core developers may already know what they want to add to the Slurm charms,
+but they are always open to new ideas and potential improvements. GitHub Discussions is
+a good place for discussing open-ended questions that pertain to the entire Charmed HPC
+project, but more focused enhancement proposal discussions can start within the issue
+tracker.
+
+Please note that not all proposals may be incorporated into the documentation. Also, please
+know that spamming the maintainers to incorporate something you want into the documentation
```

Comment:
Oop, found another section that is too docs-focused.

Reply:
I promise that I can read.